src/plugin-recall-storage/services/recall.service.ts

```diff
@@ -1,38 +1,25 @@
-import {
-  RecallClient,
-  walletClientFromPrivateKey,
-} from '../../../../js-recall/packages/sdk/dist/client.mjs';
-import { testnet } from '../../../../js-recall/packages/chains/dist/index.mjs';
-// @ts-expect-error - this is temporary
-import { CreditAccount } from '../../../../js-recall/packages/sdk/dist/credit.mjs';
-// @ts-expect-error - this is temporary
-import { ListResult } from '../../../../js-recall/packages/sdk/dist/bucket.mjs';
 import { elizaLogger, UUID, Service, ServiceType } from '@elizaos/core';
-import { parseEther, TransactionReceipt } from 'viem';
+import { testnet } from '@recallnet/chains';
+import { AccountInfo } from '@recallnet/sdk/account';
+import { ListResult } from '@recallnet/sdk/bucket';
+import { RecallClient, walletClientFromPrivateKey } from '@recallnet/sdk/client';
+import { CreditAccount } from '@recallnet/sdk/credit';
+import { Address, Hex, parseEther, TransactionReceipt } from 'viem';
 import { ICotAgentRuntime } from '../../types/index.ts';
 
-type Address = `0x${string}`;
-type AccountInfo = {
-  address: Address;
-  nonce: number;
-  balance: bigint;
-  parentBalance?: bigint;
-};
-
 type Result<T = unknown> = {
```

Comment:
note: i'll clean this up in https://github.com/recallnet/js-recall/issues/41 and make sure it's exported